tools/custom-checks/check-apache-version.ts

```diff
@@ -0,0 +1,68 @@
+  /**   * The version that will be used as the correct one when checking the sibling   * package dependency version declarations.   * If you omit this (optional) parameter then the root package.json file   * will be parsed to obtain its value at runtime.   */ readonly version?: string;
+}
+/** * Verifies that each sibling dependency is up to date with the latest version * that was released at any given time. * Note: This only checks dependency versions for the packages that are hosted * within this monorepo (Hyperledger Cactus) not dependencies in general. * * For example if the cmd-api-server package depends on the common package and * currently the project is on release v1.2.3 then the dependency declaration of * the package.json file of the cmd-api-server package should also use this * v1.2.3 version not something outdated such as v0.3.1 because that may cause * (according to our experience) strange build issues that confuse people a lot * for no good reason. * * @returns An array with the first item being a boolean indicating * 1) success (`true`) or 2) failure (`false`) */ export async function apache2LicenseChecker(
+  req: ICheckSiblingDepVersionConsistencyRequest,
+): Promise<[boolean, string[]]> {
+  const TAG = "[tools/checkapache-version.ts]";
+  const __filename = fileURLToPath(import.meta.url);
+  const __dirname = path.dirname(__filename);
+  const SCRIPT_DIR = __dirname;
+  const PROJECT_DIR = path.join(SCRIPT_DIR, "../../");
+  const correctPackage = "Apache-2.0";
+  console.log(`${TAG} SCRIPT_DIR=${SCRIPT_DIR}`);
+  console.log(`${TAG} PROJECT_DIR=${PROJECT_DIR}`);
+  if (!req) {
+    throw new RuntimeError(`req parameter cannot be falsy.`);
+  }
+  if (!req.argv) {
+    throw new RuntimeError(`req.argv cannot be falsy.`);
+  }
+  if (!req.env) {
+    throw new RuntimeError(`req.env cannot be falsy.`);
+  }
+  const globbyOpts: GlobbyOptions = {
+    cwd: PROJECT_DIR,
+    ignore: ["**/node_modules"],
+  };
+  const DEFAULT_GLOB = "*/cactus-*/package.json";
+  const pkgJsonPaths = await globby(DEFAULT_GLOB, globbyOpts);
+  const oasPaths = await globby(DEFAULT_GLOB, globbyOpts);
+  console.log(`openapi.json paths (${oasPaths.length})`);
+  console.log(`${TAG} Correct Package: ${correctPackage}`);
+  const errors: string[] = [];
+  const checks = pkgJsonPaths.map(async (pathRel) => {
+    const filePathAbs = path.join(PROJECT_DIR, pathRel);
+    const pkgJson: unknown = await fs.readJSON(filePathAbs);
+    if (typeof pkgJson !== "object") {
+      errors.push(`ERROR: ${pathRel} package.json cannot be empty.`);
+      return;
+    }
+    if (!pkgJson) {
+      errors.push(`ERROR: ${pathRel} package.json cannot be empty.`);
+      return;
+    }
+    if (!isStdLibRecord(pkgJson)) {
+      return;
+    }
+    const { license } = pkgJson;
+    if (license == correctPackage) {
+      const msg = `${TAG} PASSED: ${pathRel} the package itself correct has version ${license}. Expected ${correctPackage}`;
```

Comment:
@micoferdinand98 The log message could be shortened to `${TAG} PASSED: ${pathRel} has the correct licnese.` and then it might be that it fits into a single line (it might be still too long but I just wanted to make a point that sometimes you can compress the sentence a little without losing any information and make the code a little more readable if the line doesn't have to be broken by the formatter)